sql/core/src/test/scala/org/apache/spark/sql/execution/benchmark/MiscBenchmark.scala

```diff
@@ -21,247 +21,125 @@ import org.apache.spark.benchmark.Benchmark
 
 /**
  * Benchmark to measure whole stage codegen performance.
- * To run this:
- *  build/sbt "sql/test-only *benchmark.MiscBenchmark"
- *
- * Benchmarks in this file are skipped in normal builds.
+ * To run this benchmark:
+ * 1. without sbt: bin/spark-submit --class <this class> <spark sql test jar>
+ * 2. build/sbt "sql/test:runMain <this class>"
+ * 3. generate result: SPARK_GENERATE_BENCHMARK_FILES=1 build/sbt "sql/test:runMain <this class>"
+ *    Results will be written to "benchmarks/MiscBenchmark-results.txt".
  */
-class MiscBenchmark extends BenchmarkWithCodegen {
-
-  ignore("filter & aggregate without group") {
```

Comment:
This refactoring introduces a long function body at `runBenchmarkSuite`. In general, it's not a better direction.

Could you map each `ignore` function  into an independent function and make `runBenchmarkSuite()` invoke a series of those functions?